docs/concepts/testnets.md

```diff
@@ -0,0 +1,116 @@
+---
+id: testnets
+---
+
+# Gno Testnets
+
+This page documents all Gno.land testnets, what their properties are, and how
+they are meant to be used. For testnet configuration, visit the 
+[reference section](../reference/network-config).
+
+Gno.land testnets are categorized by 4 main points:
+- **Persistence of state**
+  - Is the state and transaction history persisted?
+- **Timeliness of code**
+  - How up-to-date are Gno language features and demo packages & realms?
+- **Intended purpose**
+  - When should this testnet be used?
+- **Versioning strategy**
+  - How is this testnet versioned?
+
+Below you can find a breakdown of each existing testnet by these categories.
+
+## Portal Loop
+Portal Loop is an always up-to-date rolling testnet. It is meant to be used as 
+a nightly build of the Gno tech stack. The home page of [gno.land](https://gno.land)
+is the `gnoweb` render of the Portal Loop testnet. 
+
+- **Persistence of state:**
+  - State is kept on a best-effort basis 
+  - Transactions that are affected by breaking changes will be discarded
+- **Timeliness of code:**
+  - Packages & realms which are available in the `examples/` folder on the [Gno
+monorepo](https://github.com/gnolang/gno) exist on the Portal Loop in matching 
+state - they are refreshed with every new commit to the `master` branch.
+- **Intended purpose**
+  - Providing access the latest version of Gno for fast development & demoing
+- **Versioning strategy**:
+  - Portal Loop infrastructure is managed within the `misc/loop` folder in the 
```

Comment:
I left a question the PR description, which is why this is the way it is:

> Since test4 will be versioned based on Gno releases, will this also include new code in the examples/ folder? How will this work? cc @zivkovicmilos